src/WacsdkCommandConfig.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Ipfs.Http;
+using OwlCore.Nomad.Kubo;
+using OwlCore.Storage.System.IO;
+
+public class WacsdkCommandConfig
+{
+    public CancellationToken CancellationToken { get; set; }
```

Comment:
We should remove this-- cancellation was passed through here for cancelling via `Console.CancelKeyPress`